pyproject.toml

```diff
@@ -113,4 +114,8 @@ message_template = "Bump to {new_version}"
 tag_template = "{new_version}"
 
 [[tool.tbump.file]]
-src = "jupyterhub-pytest-plugin/_version.py"
\ No newline at end of file
+src = "jupyterhub-pytest-plugin/_version.py"
+
+# specify plugin entrypoints
+[project.entry-points.pytest11]
+jupyterhub-spawners = "jupyterhub_spawners"
```

Comment:
Let's:
- add a link to the pytest plugin entrypoints docs
- specify the package name when listing the plugin
- be more explicitit that `jupyterhub-spawners` entrypoint refer to a plugin

```suggestion
# specify project entrypoints to make each plugin discoverable by pytest
# ref: https://docs.pytest.org/en/latest/how-to/writing_plugins.html#making-your-plugin-installable-by-others
[project.entry-points.pytest11]
jupyterhub-spawners-plugin= "jupyterhub-pytest-plugin.jupyterhub_spawners"
```